tests/test_multiiter_pos_calculation.py

```diff
@@ -0,0 +1,59 @@
+import os
+import logging
+import numpy as np
+
+import ptychonn.position
+from ptychonn.position.configs import InferenceConfigDict, RegistrationConfigDict
+from ptychonn.position.core import ProbePositionCorrectorChain
+from ptychonn.position.position_list import ProbePositionList
+
+
+logging.basicConfig(format="[%(asctime)s] %(message)s", level=logging.INFO)
+
+
+def test_multiiter_pos_calculation():
+    scan_idx = 235
+
+    configs = InferenceConfigDict(
+        reconstruction_image_path=os.path.join(
+            "data", "position", "pred_test{}".format(scan_idx), "pred_phase.tiff"
+        ),
+        random_seed=196,
+        debug=False,
+        probe_position_list=None,
+        # To start with an initial position set, do the folliwing instead:
+        # probe_position_list=ProbePositionList(position_list=arr),
+        # where arr is a [N, 2] array of probe positions in pixel.
+        central_crop=None,
+        num_neighbors_collective=4,
+        registration_params=RegistrationConfigDict(registration_method="hybrid"),
+    )
+    # One can use different values for a config key at different iterations. To do this, create dict keys in the
+    # config object with a name of "<existing_config_ket_name>_multiiter". For example:
+    # configs.__dict__['method_multiiter'] = ["serial", "collective", "collective"]
+    # configs.__dict__['hybrid_registration_tols_multiiter'] = [[0.3, 0.15], [0.15, 0.1], [0.15, 0.1]]
+    # Alternatively, you may also put these multiiter keys in a json or toml and read them.
+    configs.load_from_toml(
+        os.path.join("data", "position", "config_{}.toml".format(scan_idx))
+    )
+    print(configs)
```

Comment:
Is it possible to do something like

```python
configs = InferenceConfigDict.load_from.toml("path/to/file.toml")
```

for when the user wants to only define the params in file format?

Reply:
Added 2 static methods `from_toml` and `from_json` to do this in b093eb3.